src/Service/Terminal/Service_Terminal.rei

```diff
@@ -1,20 +1,18 @@
 open Oni_Core;
 open Oni_Extensions;
 
-module Msg: {
-  type t =
-    // TODO:
-    | Resized(unit)
-    | Updated(unit)
-    | ProcessStarted({
-        id: int,
-        pid: int,
-      })
-    | ProcessTitleSet({
-        id: int,
-        title: string,
-      });
-};
+type msg =
+  // TODO:
+  | Resized(unit)
+  | Updated(unit)
```

Comment:
_What_ has been resized and updated? Is `Resized` referring to the viewport?

Reply:
I'll remove these for now. They are placeholders for the incoming screen / rendering work.